packages/pangea-sdk/tests/integration/test_authz.py

```diff
@@ -114,56 +90,61 @@ def test_integration(self):
 
         # Check no debug
         r_check = self.authz.check(
-            resource=m.Resource(namespace=namespace_folder, id=folder1),
+            resource=m.Resource(type=type_folder, id=folder1),
             action="reader",
-            subject=m.Subject(namespace=namespace_user, id=user2),
+            subject=m.Subject(type=type_user, id=user2),
         )
 
         self.assertIsNotNone(r_check.result)
+        assert r_check.result
```

Comment:
What's the difference with de assert above? Does it check empty strings too?

Reply:
This is for type-checking, because `self.assertIsNotNone()` is not a type guard in mypy's eyes.